__test__/helper.js

```diff
@@ -0,0 +1,26 @@
+import webdriver from 'selenium-webdriver'
+
+const capabilities = {
+  'os_version' : '10',
+  'resolution' : '1920x1080',
+  'browserName' : 'Chrome',
+  'browser_version' : 'latest-beta',
```

Comment:
Do we want to do our tests with a beta version of Chrome? I would suggest to take the latest stable version.